packages/lesswrong/components/common/ContentItemBody.tsx

```diff
@@ -291,6 +301,10 @@ class ContentItemBody extends Component<ContentItemBodyProps,ContentItemBodyStat
   }
 }
 
+const addNofollowToHTML = (html: string): string => {
+  return html.replace('<a ', '<a rel="nofollow" ')
```

Comment:
I think you need `replaceAll`. Can test with [this comment](http://localhost:3000/posts/FWdgCyL5FDpPRRRu8/post-102?commentId=WwTDRtooEf3PM5rn2)